pylib/Tools/Build/Autotools.py

```diff
@@ -197,6 +199,16 @@ def execute(self, log, keyvals, testDef):
                     # this is a multistep operation, and so we need to
                     # retain the output from each step in the log
                     log['autogen'] = (stdout, stderr)
+
+                    # Use mpi4py to get MPI information for IUDatabase plugin
+                    log['mpi_name'] = MPI.get_vendor()[0]
```

Comment:
Given that mpi4py is open source, can we look at its license and (if compatible) simply extract the content of this function? It would be preferable to including the entire mpi4py library just to call this one thing.